JS/Lesson27/gulcin/editPost/styles.css

```diff
@@ -0,0 +1,47 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f2f2f2;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+#post-form {
+    width: 30%;
+    padding: 16px;
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    height: 50%;
```

Comment:
On a small screen the form and inputs shrink a lot and it's not even possible to see content of the inputs. In reality, if you set width of an element, you probably don't want to set it's height at all. The exception is a very static element (e.g. icon) that should always be the sime size no matter what.
```suggestion
```